### PR TITLE
sqldiff: do not use absolute sqlite3 path in test, fixes test for Linux

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/sqldiff.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def install
     dbpath = testpath/"test.sqlite"
     sqlpath = testpath/"test.sql"
     sqlpath.write "create table test (name text);"
-    system "/usr/bin/sqlite3 #{dbpath} < #{sqlpath}"
+    system "sqlite3 #{dbpath} < #{sqlpath}"
```
Comment (carlocab) [inline Formula/sqldiff.rb:34]:
```suggestion
    sqlite3 = DevelopmentTools.locate("sqlite3")
    system "#{sqlite3} #{dbpath} < #{sqlpath}"
```

Comment (iMichka) [reply Formula/sqldiff.rb:34]: We don't use this in any formula. I think we should have an audit that enforces the usage of `DevelopmentTools.locate` if wanted.

Comment (Bo98) [reply Formula/sqldiff.rb:34]: It's only necessary if nothing is found in the path. And, to be honest, I can't think of a scenario where that would be the case.

Comment (iMichka) [reply Formula/sqldiff.rb:34]: So my PR is fine then? :D

**Review by carlocab — APPROVED** [review]

**Pull request merged**